src/mantine-core/src/components/FloatingTooltip/FloatingTooltip.tsx

```diff
@@ -0,0 +1,220 @@
+      gutter,
+      color,
+      radius,
+      disabled,
+      position,
+      transition,
+      transitionDuration,
+      zIndex,
+      transitionTimingFunction,
+      width,
+      wrapLines,
+      positionDependencies,
+      withinPortal,
+      tooltipRef,
+      tooltipId,
+      classNames,
+      styles,
+      onMouseLeave,
+      onMouseEnter,
+      ...others
+    } = useMantineDefaultProps('FloatingTooltip', defaultProps, props);
+
+    const { classes, cx } = useStyles(
+      { color, radius },
+      { classNames, styles, name: 'FloatingTooltip' }
+    );
+    const openTimeoutRef = useRef<number>();
+    const closeTimeoutRef = useRef<number>();
+    const [_opened, setOpened] = useState(false);
+    const { ref: mouseRef, x, y } = useMouse();
+    const visible = (typeof opened === 'boolean' ? opened : _opened) && !disabled;
+    const [referenceElement, setReferenceElement] = useState(null);
+    const mergedRefs = useMergedRef(ref, setReferenceElement, mouseRef);
+    const coordinates = useMemo(() => {
+      switch (position) {
+        case 'top':
+          return { x: x - estimatedCursorSize, y: y - estimatedCursorSize };
+        case 'left':
+          return { x: x - estimatedCursorSize * 2, y: y };
+        case 'right':
+          return { x: x + estimatedCursorSize / 2, y: y };
+        case 'bottom':
+          return { x: x - estimatedCursorSize, y: y + estimatedCursorSize };
+        default:
+          return { x: x || 0, y: y || 0 };
+      }
+    }, [x, y]);
+
+    const handleOpen = () => {
+      window.clearTimeout(closeTimeoutRef.current);
+
+      if (openDelay !== 0) {
+        openTimeoutRef.current = window.setTimeout(() => {
+          setOpened(true);
+        }, openDelay);
+      } else {
+        setOpened(true);
+      }
+    };
+
+    const handleClose = () => {
+      window.clearTimeout(openTimeoutRef.current);
+
+      if (closeDelay !== 0) {
+        closeTimeoutRef.current = window.setTimeout(() => {
+          setOpened(false);
+        }, closeDelay);
+      } else {
+        setOpened(false);
+      }
+    };
+
+    useEffect(
+      () => () => {
+        window.clearTimeout(openTimeoutRef.current);
+        window.clearTimeout(closeTimeoutRef.current);
+      },
+      []
+    );
+
+    return (
+      <Box<'div'>
+        className={cx(classes.root, className)}
+        onMouseEnter={(event) => {
+          handleOpen();
+          typeof onMouseEnter === 'function' && onMouseEnter(event);
+        }}
+        onMouseLeave={(event) => {
+          handleClose();
+          typeof onMouseLeave === 'function' && onMouseLeave(event);
+        }}
+        onFocusCapture={handleOpen}
+        onBlurCapture={handleClose}
+        ref={mergedRefs}
+        {...others}
+      >
+        <Popper
+          referenceElement={referenceElement}
+          transitionDuration={transitionDuration}
+          transition={transition}
+          mounted={visible}
+          position="top"
```

Comment:
These Popper props are hard-coded here, as they are a bit misleading. They are all relative to the position of the reference element, which isn't what we want to do.